### DISCUSSION TITLE
Mocking onetable using jest in nodejs clarification/help

### DISCUSSION THREAD

**Discussion by waelhu** [sensedeep/dynamodb-onetable#219] [Q&A] [answered]
Can you confirm how to mock the Table class in jest and its underlying methods for the get/create/fetch etc.
I have some code in a handler that creates a user record out of my dynamodb table but unable to mock that behaviour using Jest. I couldn't find the specific example in /test

Thank you

**Comment by mobsense** [comment, marked as answer]
Sorry I missed this question.

We don't mock the Table class in jest. We just use it fully. See /test in the source where we run a local dynamo instance and use jest for our unit tests. Let me know if I'm misunderstanding your question.

**Comment by onhate** [comment]
@waelhu see https://www.npmjs.com/package/jest-dynalite
